lib/rpc/SingleJsonRpcProvider.ts

```diff
@@ -341,6 +365,8 @@ export class SingleJsonRpcProvider extends StaticJsonRpcProvider {
         // Update latency stat
         this.updateLatencyStat(newState)
       }
+      this.lastDbSyncTimestampInMs = Date.now()
+      this.log.debug('Successfully synced with DB and updated states')
```

Comment:
nit: you can put log debug into `logDbSyncSuccess()`, although the debug statement itself seems unnecessary because you are logging the success metric.

Reply:
The logging is useful when looking through the logs after setting the logging level to `INFO`. I want to make the `logDbSyncSuccess()` specific to metric reporting. So having this seemingly extra two calls is fine I guess.